tests/engine/daemon/test_execmanager.py

```diff
@@ -164,7 +167,8 @@ def test_upload_local_copy_list(
     calc_info.local_copy_list = [[folder.uuid] + local_copy_list]
 
     with node.computer.get_transport() as transport:
-        execmanager.upload_calculation(node, transport, calc_info, fixture_sandbox)
+        runner = get_manager().get_runner()
+        runner.loop.run_until_complete(execmanager.upload_calculation(node, transport, calc_info, fixture_sandbox))
```

Comment:
why is this needed now?

Reply:
Because `execmanager.upload_calculation` is now a async function.. this way we can call it in a sync test.

Reply:
What happens if you use the old way? The test just passes and continues before finishing the command?

Reply:
I think it is very tricky to mix up the async programming and sync function, it is in general a very hard problem. This looks to me the `runner.loop.run_until_complete` will block the running of the task until it complete so give no benefit after making these methods async. Is the `create_task` the correct thing to use?

Reply:
Okay, I just asked Ali offline. This is only for tests and only for test the functionality of the implementation is correct. The async behaviors of four operations working together is not the purpose here.